.github/CONTRIBUTING.md

```diff
@@ -28,6 +28,6 @@ fit within the scope of any of the existing doc fix projects.
 
 ## Submission Guidelines
 
-Read more about our [submission guidelines](https://github.com/IBM/beacon-for-ibm-dotcom/blob/main/docs/submission-guidelines.md).
+Read more about our [submission guidelines](https://github.com/IBM/beacon/blob/main/docs/submission-guidelines.md).
```

Comment:
The repo name won't change though, right?

Reply:
I figured we'd keep it consistent, npm-style. You want the repo name to stay the same?